From 03529794c6e16e14b11430f38d5552dcf06d4b85 Mon Sep 17 00:00:00 2001
From: willson-chen <willson.chenwx@gmail.com>
Date: Fri, 10 Apr 2020 15:24:09 +0800
Subject: [PATCH] src/config.h.cmake: replace #cmakedefine01 with #define in
 macro ENABLE_GD_FOORMATS

Although we have a document describ the avaiable options
are ENABLE_XXX=1/0, there must some one are familar with
ENABLE_XXX=ON/OF.  If ENABLE_GD_FORMATS=ON is used, it will
result that GD formats is not enabled actually.
---
 src/config.h.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/config.h.cmake b/src/config.h.cmake
index 54f3a528..836be470 100644
--- a/src/config.h.cmake
+++ b/src/config.h.cmake
@@ -4,7 +4,7 @@
 #cmakedefine BGDWIN32
 
 /* Whether to support gd image formats */
-#define ENABLE_GD_FORMATS @ENABLE_GD_FORMATS@
+#cmakedefine01 ENABLE_GD_FORMATS
 
 /* Define to 1 if you have the <dirent.h> header file. */
 #cmakedefine HAVE_DIRENT_H
